conf.go

```diff
@@ -52,6 +52,9 @@ type ServerConf struct {
 	ForwardedHeaders     map[string]bool
 	KeepAppLive          bool
 	PingInterval         time.Duration
+	PreviewImage         string
+	PreviewTitle         string
+	PreviewDescription   string
```

Comment:
Make these part of the https://github.com/h2oai/wave/blob/94bf37f538719ad279a41b0df1b452623d61aaa4/conf.go#L71C7-L71C7 as well to expose new options to the users.

Also document [here](https://github.com/h2oai/wave/blob/main/website/docs/configuration.md).

Reply:
I'll get started on this as well as a demo :)

Reply:
Hi Martin, 

I started working on the demo. Using Meta's Open Graph debugger, it can read the custom title and description, but there are some issues with the image.
<img width="986" alt="image" src="https://github.com/h2oai/wave/assets/87551908/231c8d77-1371-4115-9007-ce894743042e">
<img width="988" alt="image" src="https://github.com/h2oai/wave/assets/87551908/b26a2e97-c097-405a-9255-84da6bc5c2a6">

I did some more digging and there are some guidelines for adding images to preview links
https://developers.facebook.com/docs/sharing/best-practices/#precaching

How should I approach this?

Reply:
The warning says all it needs are `og:image:height` and `og:image:width` props. I would first try it out to see if they help. If they do, the next step would be to validate passed image dimensions and:

* If the image does not adhere to the recommended ratio or is too small, log warning and use the actual dimensions for the respective props.
* If the image ratio is good and >= required dimensions, use the copy of the image resized to the expected dimensions.

Hope that makes sense.

Reply:
Yes that makes sense I'll get started on it!